src/fe_window.cpp

```diff
@@ -122,6 +126,18 @@ void FeWindow::onCreate()
 	sf::RenderWindow::onCreate();
 }
 
+void FeWindow::display()
+{
+	// Starting from Windows Vista all non fullscreen window modes
+	// go through DWM, so we have to flush here to sync to the DMW's v-sync
+	// to avoid stuttering.
+#if defined(SFML_SYSTEM_WINDOWS) && !defined(WINDOWS_XP)
+	if ( m_win_mode != FeSettings::Fullscreen )
+		DwmFlush();
```

Comment:
Hi there, I'm finally getting to reviewing your pull request now.  First question: DwmFlush() looks to be a direct x function.  Did you try out the opengl functions glFlush() and/or glFinish() before landing on using DwmFlush()?  glFlush glFinish look like they are the opengl equivalent of DwmFlush (?)

I'd really prefer sticking with opengl if we can...

Reply:
It's not really an equivalent from what I've read. DWM stands for Desktop Window Manager and since Vista all non-fullscreen windows go through it no matter what graphics API you are using. I've tried glFlush() and it wasn't doing anything, glFinish() also is not reliable as per doccumentation. It says it's up to the driver to determine whether to flush/finish or not. By syncing to DWM you are basicly wait for the window compositor signalling it finished processing all the desktop windows and it's ready for a vsync.